### DISCUSSION TITLE
关于 Kraken 受限于 Flutter 页面级混合开发，不能支持 View 级别嵌入到 native

### DISCUSSION THREAD

**Discussion by devjiangzhou** [openkraken/kraken#994] [General]
前几天看到支付宝的动态化方案，https://xie.infoq.cn/article/ad93ecc7dc7d066db60d064ac

**Comment by andycall** [comment]
把 flutter 当作 view 嵌入到 native 就可以了。

https://docs.flutter.dev/development/add-to-app

**Reply by devjiangzhou** [reply]
嗯，是的，标题没起好，其实想表达 Feed 流这种情况下的嵌入

**Reply by andycall** [reply]
那为什么不直接用 flutter + kraken 实现 feed 流
